test.ts

```diff
@@ -1,12 +1,6 @@
 import { udd, UddOptions, UddResult } from "./mod.ts";
 import { RegistryCtor } from "./registry.ts";
-import {
-  assert,
-  assertEquals,
-  assertThrowsAsync,
```

Comment:
a statement of intent 😆

Reply:
Sorry but I did nothing but follow `deno fmt`....
Why don't you add `// deno-fmt-ignore` above the line?